CHANGELOG.md

```diff
@@ -144,9 +135,21 @@ This package is just re-released since version 5.1.7 had a breaking change.
 - <!-- 13 --> [docs] Explicitly list demos of unstyled components (#25900) @eps1lon
 - <!-- 09 --> [docs] Fix global styles leaking on different pages (#25855) @mnajdova
 - <!-- 04 --> [docs] Expose heading links in a11y tree (#25861) @eps1lon
-- <!-- 16 --> [l10n] Add Bengali (bnBD) locale (#25841) @Knoxo
 - <!-- 56 --> [Table] Improve description of TablePagination.rowsPerPageOptions (#25982) @kevinlul
 
+### Core
+
+- <!-- 54 --> [core] Fix wrong imports to '@material-ui/styles' (#25984) @mnajdova
+- <!-- 52 --> [core] Ensure props spreading works as expected (#25939) @oliviertassinari
+- <!-- 47 --> [core] Batch small changes (#25968) @oliviertassinari
+- <!-- 45 --> [core] Fix various too wide `classes` types (#25917) @eps1lon
```

Comment:
part of `@material-ui/core`. You should include the affected components (listed in the PR description).